server/autolinkplugin/plugin.go

```diff
@@ -131,7 +141,23 @@ func (p *Plugin) ProcessPost(c *plugin.Context, post *model.Post) (*model.Post,
 
 		return true
 	})
-	post.Message = postText
+	if post.Message != postText {
+		isBot, appErr := p.isBotUser(post.UserId)
```

Comment:
It seems that if there is a "system failure" in the process of modifying the message, the right thing to do is to pass it through as is. 0/5 `if isBot || appErr != nil {...`

Reply:
What do you mean by `in the process of modifying the message`? ` isBotUser` does not modify the message.